src/foundry/foundry.js/config.d.ts

```diff
@@ -1463,304 +1177,188 @@ declare global {
      * Configuration for the AmbientSound embedded document type and its representation on the game Canvas
      */
     AmbientSound: {
-      /**
-       * @defaultValue `AmbientSoundDocument`
-       */
+      /** @defaultValue `AmbientSoundDocument` */
       documentClass: ConfiguredDocumentClassOrDefault<typeof AmbientSoundDocument>;
 
-      /**
-       * @defaultValue `AmbientSound`
-       */
+      /** @defaultValue `AmbientSound` */
       objectClass: ConfiguredObjectClassOrDefault<typeof AmbientSound>;
 
-      /**
-       * @defaultValue `SoundsLayer`
-       */
+      /** @defaultValue `SoundsLayer` */
       layerClass: ConstructorOf<SoundsLayer>;
-
-      /**
-       * @defaultValue `AmbientSoundConfig`
-       */
-      sheetClass: ConstructorOf<AmbientSoundConfig>;
     };
 
     /**
      * Configuration for the Combatant embedded document type within a Combat document
      */
     Combatant: {
-      /**
-       * @defaultValue `Combatant`
-       */
+      /** @defaultValue `Combatant` */
       documentClass: ConfiguredDocumentClassOrDefault<typeof Combatant>;
-
-      /**
-       * @defaultValue `CombatantConfig`
-       */
-      sheetClass: ConstructorOf<CombatantConfig>;
     };
 
     /**
      * Configuration for the Drawing embedded document type and its representation on the game Canvas
      */
     Drawing: {
-      /**
-       * @defaultValue `DrawingDocument`
-       */
+      /** @defaultValue `DrawingDocument` */
       documentClass: ConfiguredDocumentClassOrDefault<typeof DrawingDocument>;
 
-      /**
-       * @defaultValue `Drawing`
-       */
+      /** @defaultValue `Drawing` */
       objectClass: ConfiguredObjectClassOrDefault<typeof Drawing>;
 
-      /**
-       * @defaultValue `DrawingsLayer`
-       */
+      /** @defaultValue `DrawingsLayer` */
       layerClass: ConstructorOf<DrawingsLayer>;
-
-      /**
-       * @defaultValue `DrawingConfig`
-       */
-      sheetClass: ConstructorOf<DrawingConfig>;
     };
 
     /**
      * Configuration for the MeasuredTemplate embedded document type and its representation on the game Canvas
      */
     MeasuredTemplate: {
       defaults: {
-        /**
-         * @defaultValue `53.13`
-         */
+        /** @defaultValue `53.13` */
         angle: number;
 
-        /**
-         * @defaultValue `1`
-         */
+        /** @defaultValue `1` */
         width: number;
       };
 
       types: {
-        /**
-         * @defaultValue `'Circle'`
-         */
+        /** @defaultValue `'Circle'` */
         circle: string;
 
-        /**
-         * @defaultValue `'Cone'`
-         */
+        /** @defaultValue `'Cone'` */
         cone: string;
 
-        /**
-         * @defaultValue `'Rectangle'`
-         */
+        /** @defaultValue `'Rectangle'` */
         rect: string;
 
-        /**
-         * @defaultValue `'Ray'`
-         */
+        /** @defaultValue `'Ray'` */
         ray: string;
       } & Partial<Record<string, string>>;
```

Comment:
Not sure if it still makes sense to keep this extendable. The possible types are defined in `CONST` anyways... So we might just remove the intersection completely.